client/src/view/container/session-delete-window/DeleteSessionWindow.tsx

```diff
@@ -0,0 +1,15 @@
+import { deleteSessionController } from "Configuration"
+import { ViewSession } from "controller/model/ViewSession"
+import { useDeleteSessionWindow } from "./useDeleteSessionWindow"
+
+interface DeleteSessionWindowProps {
+    id: string,
+    setSession: (session: ViewSession | undefined) => void
+}
+
+export const DeleteSessionWindow = ({id, setSession} : DeleteSessionWindowProps) => {
+
+    const deleteSession = useDeleteSessionWindow(deleteSessionController, setSession)
+
+    return <button onClick={() => {deleteSession(id)}}>Quit</button>
```

Comment:
jeigu nesiruoši dėti čia daugiau logikos, tai gal mažiau complicated būtų buvę palikti šitą logiką pačiame GameWindow?

Reply:
Kai pradėsiu su Material UI, šitą tikriausiai keisiu į modalinį langa, dėl to palikau kaip atskirą komponentą